src/blockchains/erc20/lib/contract_overwrite.js

```diff
@@ -42,10 +42,12 @@ function changeContractAddresses(events, contractsOverwriteArray) {
  */
 function editAddressAndAmount(event, contractOverwrite) {
   const multiplier = contractOverwrite.mapAddressToMultiplier[event.contract];
-  if (!multiplier) {
+  if (multiplier === undefined) {
     throw new Error('Event contract does not match expected');
   }
-  event.contract = contractOverwrite.newContract;
+  const valueBigNumber = new BigNumber(event.value.toString());
+  const updatedValue = valueBigNumber.times(multiplier).integerValue(BigNumber.ROUND_FLOOR);
```

Comment:
`integerValue` seems not necessary because we dont have floating point values here, do we?